docs/enterprise-config.md

```diff
@@ -0,0 +1,57 @@
+# Enterprise configuration defaults
+
+Git Credential Manager Core (GCM Core) can be configured using multiple
+different mechanisms. In order of preference, those mechanisms are:
+
+1. [Environment variables](environment.md)
+2. [Standard Git configuration files](configuration.md)
+   1. Repository/local configuration (`.git/config`)
+   2. User/global configuration (`$HOME/.gitconfig` or `%HOME%\.gitconfig`)
+   3. Installation/system configuration (`etc/gitconfig`)
+3. Operating system specific configuration store
+
+This model largely matches what Git itself supports, namely environment
+variables that take precedence over Git configuration files.
+
+The addition of the operating system specific configuration store enables system
+administrators to configure defaults for many settings that may be required in
+an enterprise or corporate setting.
+
+---
+
+**Important:** We believe the user should _always_ be at liberty to configure
+Git and GCM exactly as they wish. By prefering environment variables and Git
+configuration files over OS store values, these only act as _default values_
+that can always be overriden by the user in the usual ways.
```

Comment:
I appreciate this caveat. Could we stick it in a `###`-header so we can link directly to it when someone complains either that:

1. their system admin gets too much control over their machine (they don't), or
2. the system admin complains that users are overriding their settings.

Reply:
Done!